subtests/docker_cli/meta/meta.py

```diff
@@ -0,0 +1,62 @@
+r"""
+Summary
+-------
+
+Meta-tests: docker packaging, installation, setup. Nothing here
+actually runs docker.
+
+Operational Summary
+-------------------
+
+#. Run 'file' against docker binaries, make sure output includes 'not stripped'
+"""
+
+import os
+import magic
+from dockertest import subtest
+from dockertest.subtest import SubSubtest
+
+
+class meta(subtest.SubSubtestCaller):
+
+    """ SubSubtest caller """
+
+
+class meta_base(SubSubtest):
+    pass
+
+
+class debuginfo_present(meta_base):
+    """
+    bz1280068 - make sure docker binary is shipped with debug info.
+    We simply use libmagic to check for us. Note that this may fail
+    if/when it becomes possible to ship debuginfo as a separate rpm.
+    """
+
+    def initialize(self):
+        super(debuginfo_present, self).initialize()
+        # The python-magic package that ships with RHEL is clunky, obsolete,
+        # undocumented, and bears no relation to any currently known magic.py
+        # in other distros or via pip. Refer to:
+        #     https://stackoverflow.com/questions/25286176
+        self.sub_stuff['magic'] = magic.open(magic.MAGIC_NONE)
+        self.sub_stuff['magic'].load()
+
+    def run_once(self):
+        super(debuginfo_present, self).run_once()
+        found_binary = False
+        # Another way to search for DWARF might be:
+        #    readelf -S /usr/bin/docker-current |grep -i debug_info
+        for binfile in ['docker-current', 'docker-latest']:
+            path = os.path.join('/usr/bin', binfile)
+            if os.path.exists(path):
+                found_binary = True
```

Comment:
The purpose was to make sure that we found at least one docker binary. Maybe I should've named the variable `found_any_binary`. But on reflection I'm not sure it actually serves any useful purpose; let me consider removing it.